CJBCheatsMenu/i18n/th.json

```diff
@@ -0,0 +1,242 @@
+{
+    /*********
+    ** Main translations
+    *********/
+    // mod name shown in the menu
+    "mod-name": "CJB Cheats Menu",
+
+    // tab names
+    "tabs.player-and-tools": "ผู้เล่น & เครื่องมือ",
+    "tabs.farm-and-fishing": "การทำฟาร์ม & การตกปลา",
+    "tabs.skills": "ทักษะ",
+    "tabs.weather": "สภาพอากาศ",
+    "tabs.relationships": "ความสัมพันธ์",
+    "tabs.warp": "ตำแหน่งวาร์ป",
+    "tabs.time": "เวลา",
+    "tabs.advanced": "ตั้งค่าขั้นสูง",
+    "tabs.controls": "ตัวควบคุม",
+
+
+    /*********
+    ** Player & Tools tab
+    *********/
+    // player
+    "player.title": "ĕÇÅÐ¤Ãڌà˕¹", // ตัวละครผู้เล่น
```

Comment:
Some of these translations seem to have corrupted text?

Reply:
@Pathoschild We matched Thai characters with [unicode tables](https://unicode-table.com/en/) to create **LooseSprites/font_bold.png** files. which works well with stardew.

![](https://github.com/ellipszist/StardewMods/raw/master/_encodes/StardewValley/assets/Content/LooseSprites/font_bold_Redux.png)

Ref: https://github.com/ellipszist/StardewMods/tree/master/_encodes/StardewValley/assets/Content/LooseSprites